tests/unit/clients/test_google_drive_client.py

```diff
@@ -156,6 +161,13 @@ def test_scandir_folder(self, mocker, folder_descriptor, url):
         kwargs = lister.mock_calls[0][2]
         assert kwargs["url_base"] == "gdrive:/My Drive/folder/"
 
+    def test_remove(self, client, file_descriptor):
+
+        client.remove()
+
+        kwargs = client._service.files.return_value.delete.mock_calls[0][2]
+        kwargs["fileId"] = file_descriptor.id_
```

Comment:
missing assert from other PR sneaking in here

Reply:
rebasing!